src/traces/scatter/attributes.js

```diff
@@ -363,6 +364,7 @@ module.exports = {
             'marker color, or marker line color, whichever is available.'
         ].join(' ')
     },
+    fillpattern: pattern,
```

Comment:
Since this changes the `plot_schema`, please run the commands below and push:
```bash
npm run schema && git add test/plot-schema.json && git commit -m "update plot-schema diff"
```

Reply:
updated in 22e88c1e8a4fa3e5658152e3bdaa28c338dff5c3